lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -191,3 +193,22 @@ def psfont(*args, **kwargs):
     ax.text(0.5, 0.5, 'hello')
     with tempfile.TemporaryFile() as tmpfile, pytest.raises(ValueError):
         fig.savefig(tmpfile, format='pdf')
+
+
+def test_pdf_savefig_when_color_is_none():
+    backup_params = mpl.rcParams.copy()
+    mpl.rcParams.update(mpl.rcParamsDefault)
+    plt.subplot()
+    plt.axis('off')
+    plt.plot(np.sin(np.linspace(-5, 5, 100)), 'v', c='none')
+    try:
+        plt.savefig("figure.pdf", format='pdf')
+    except Exception:
+        pytest.fail("Failed to save pdf")
+    plt.savefig("figure.eps", format='eps')
+    result = compare_images('figure.pdf', 'figure.eps', 0)
+    assert result is None
+    from os import remove
```

Comment:
The following line can just be `os.remove('figure.eps')`, and then this import line isn't needed.